envs/example/defaults-2.0.yml

```diff
@@ -646,6 +646,9 @@ serverspec:
 inspec:
   enabled: False
 
+horizon:
+  customize: false
```

Comment:
Im doing this here so we can use that variable in keystone role. Ill add more code to answer that part